TripHouse/index.js

```diff
@@ -84,10 +84,18 @@ function insertHomesItem(homesItemCreated) {
 }
 
 function createInitialItems() {
-  for (let i = 0; i < data.length; i++) {
-    const homesItemCreated = createHomesItem(data, i);
-    insertHomesItem(homesItemCreated);
-  }
+  fetch("https://fe-student-api.herokuapp.com/api/hotels/popular")
+    .then((response) => response.json())
+    .then((data) => {
+      const dataFinal = data;
+      for (let i = 0; i < dataFinal.length; i++) {
```

Comment:
Try to Use array methods (map, ForEach) instead of FOR construction. It will make your code more readable